packages/jupyter-ai/src/handler.ts

```diff
@@ -219,4 +197,32 @@ export namespace AiService {
       completions_model_provider_id: modelId
     });
   }
+
+  export async function getModelParameters(
+    modelId?: string,
+    provider?: string
+  ): Promise<any> {
```

Comment:
Would it be possible to improve type-safety and provide developers that might work on this code later with type hints by defining and using a more precise type?

Reply:
@dlqqq